themes/src/main/resources/theme/base/login/resources/js/webauthnAuthenticate.js

```diff
@@ -0,0 +1,72 @@
+import { base64url } from "../../../common/keycloak/node_modules/rfc4648/lib/rfc4648.js";
+
+window.authenticateByWebAuthn = (isUserIdentified, challenge, userVerification, rpId, createTimeout, errmsg) => {
+    if (!isUserIdentified) {
+        doAuthenticate([], challenge, userVerification, rpId, createTimeout, errmsg);
+        return;
+    }
+    checkAllowCredentials(challenge, userVerification, rpId, createTimeout, errmsg);
+}
+
+function checkAllowCredentials(challenge, userVerification, rpId, createTimeout, errmsg) {
+    let allowCredentials = [];
+    let authn_use = document.forms['authn_select'].authn_use_chk;
+    if (authn_use !== undefined) {
+        if (authn_use.length === undefined) {
+            allowCredentials.push({
+                id: base64url.parse(authn_use.value, {loose: true}),
+                type: 'public-key',
+            });
+        } else {
+            for (let i = 0; i < authn_use.length; i++) {
```

Comment:
Rather than using an index you can use a `for..of` loop here instead.

```js
for (const entry of authn_use) {
}
```

Reply:
I fixed the codes as you suggested.